src/patternfly/components/VerticalNav/styles.scss

```diff
@@ -23,11 +23,12 @@
   --pf-c-vertical-nav__link--disabled--after--BackgroundColor:  transparent;
 
   // Vertical nav link colors
-  --pf-c-vertical-nav__link--Color:             var(--pf-global--Color--dark-100);
-  --pf-c-vertical-nav__link--hover--Color:      var(--pf-global--Color--active);
-  --pf-c-vertical-nav__link--focus--Color:      var(--pf-global--Color--active);
-  --pf-c-vertical-nav__link--active--Color:     var(--pf-global--Color--active);
-  --pf-c-vertical-nav__link--disabled--Color:   var(--pf-global--Color--disabled);
+  --pf-c-vertical-nav__link--Color:               var(--pf-global--Color--dark-100);
+  --pf-c-vertical-nav__link--hover--Color:        var(--pf-global--Color--dark-100);
+  --pf-c-vertical-nav__link--focus--Color:        var(--pf-global--Color--dark-100);
+  --pf-c-vertical-nav__link--active--FontWeight:  var(--pf-global--FontWeight--semi-bold);
```

Comment:
@kybaker should we make all links on the vertical nav semibold instead of just the active states. By doing this will be making the font shift a bit when active.

Reply:
I am okay with the shifting on select. She should not have any text change on hover. That will make the change less jarring.